TideApp/TideApp/Scenes/LocationsView/LocationsInfoView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  LocationsInfoView.swift
+//  TideApp
+//
+//  Created by John Sanderson on 14/05/2021.
+//
+
+import SwiftUI
+
+struct LocationInfoView: View {
+
+  let weatherInfo: [WeatherInfo]
+
+  var body: some View {
+    NavigationView {
+      ScrollView(showsIndicators: false) {
+        LazyVStack(alignment: .leading) {
+          ForEach(weatherInfo, id: \.locationName) { info in
+            LocationButton(weatherInfo: info, type: .current)
+          }
+        }
+        .padding([.top, .bottom])
+      }
+      .navigationTitle("My Locations")
+      .navigationBarItems(trailing: Image(systemName: "magnifyingglass"))
```

Comment:
could be done in another ticket but we could add images to the stylesheet as an enum or something so we don't have to type the name, makes it easier if the name changes or something as well

Reply:
to be honest we dont even need this yet so I'll just remove it for the time being